turbinia/lib/google_cloud.py

```diff
@@ -14,31 +13,61 @@
 # limitations under the License.
 """Google Cloud resources library."""
 
-from __future__ import unicode_literals
 import logging
+import traceback
 
-from google.cloud import error_reporting
-from google.cloud import exceptions
+import google.auth
```

Comment:
mmmh,google.auth was removed in the poetry lockfile

Reply:
as per quick sync, lgtm

Reply:
google.auth brought in by libcloudforensics dependencies, as long as we use libcf we don't need to be explicit about these.